src/LowercaseSentenceTokenizer.java

```diff
@@ -30,7 +30,30 @@ public class LowercaseSentenceTokenizer implements Tokenizer {
    */
   public List<String> tokenize(Scanner scanner) {
     // TODO: Implement this function to convert the scanner's input to a list of words and periods
-    return null;
+
+    List<String> tokens = new ArrayList<>();
+    while (scanner.hasNext()){
+    
+      String token = scanner.next();
+      
+      if (token != null){
+        
+        
+        if (token.endsWith(".")){
+          //String period = String.join("." + token);
+          String noPeriod = token.replace(".","");
+          tokens.add(noPeriod);
+          tokens.add(".");
```

Comment:
Nice logic! But I think there might be a slight bug. What would happen if the token was something like `sPoNgE.` (note the capitalization and ending period). How could you fix this?